build.assets/tooling/cmd/changelog/README.md

```diff
@@ -0,0 +1,81 @@
+# changelog
+
+This script generates a changelog for a release.
+
+## Usage
+
+```
+usage: changelog [<flags>]
+
+Flags:
+  --[no-]help              Show context-sensitive help (also try --help-long and --help-man).
+  --baseBranch=BASEBRANCH  The base release branch to generate the changelog for. It will be of the form branch/v* ($BASE_BRANCH)
+  --baseTag=BASETAG        The tag/version to generate the changelog from. It will be of the form vXX.Y.Z, e.g. v15.1.1 ($BASE_TAG)
+```
+
+It can optionally take two input variables: BASE_BRANCH: The base release
+branch to generate the changelog for. It will be of the form "branch/v*".
+BASE_TAG: The tag/version to generate the changelog from. It will be of the
+form "vXX.Y.Z", e.g. "v15.1.1"
+
+
+If neither are provided, the values will be automatically determined if
+possible:
+* The current branch will be used as the base branch if it matches the
+  pattern branch/v*
+* If the current branch is forked from a base branch, the base branch will be
+  used. e.g. if you create release/15.1.1 from branch/v15, the branch/v15
+  will be the base branch.
+* The base tag will be determined by running "make print-version" from the
+  root of the repo.
+
+
+Enterprise PR changelogs will be listed after the OSS changelogs. You need to
+determine if it is suitable to include them. If you do, remove the markdown
+link from each changelog when adding the changelog to CHANGELOG.md. These
+links wont work for the general public. Keep the links when adding the
+changelog to the release PR so that the enterprise PRs will link to the
+release PR.
+
+
+A changelog line may be marked with "NOCL:". This means there was no
+"Changelog:" line on the PR, and the PR title was used instead. You will
+likely need to reword these.
```

Comment:
Can we just remove this logic? The bot requires a changelog line on the PR unless the `no-changelog` label is present.

Reply:
Yeah didn't realize this wasn't necessary anymore but looking at the data you're right. Removed this functionality.